### PR TITLE
Adding the README.md File.

### PR DESCRIPTION
**Pull request by rkraja2751**

### CODE REVIEW

**Comment by rkraja2751** [comment]
Please do check this.

File: README.md [hunk 1]
```diff
@@ -1 +1,14 @@
-# Operations Research Lab (MA39014), IIT Kharagpur coursework.
```
Comment (dibyadas) [inline README.md:1]: Keep this line

File: README.md [hunk 2]
```diff
@@ -1 +1,14 @@
...
+**OPERATIONS RESEARCH LAB**
```
Comment (dibyadas) [inline README.md]: Delete this

**Review by dibyadas — CHANGES_REQUESTED** [review]

**Comment by rkraja2751** [comment]
All changes done.

**Comment by dibyadas** [comment]
Nice work @rkraja2751 !

**Pull request merged**